### PR TITLE
Added new items to stats and collect command

### PR DESCRIPTION
**Pull request by roskyfrosky**
Added new items to stats and collect command

### CODE REVIEW

File: stats_config.yaml [hunk 1]
```diff
@@ -128,3 +128,15 @@
   query: .logGroups|length
   source: logs-describe-log-groups
   verbose: true
+- name: ACM Certificates 
+  query: .CertificateSummaryList|length
+  source: acm-list-certificates  
+  verbose: true
+- name: DynamoDB Tables 
+  query: .TableNames|length
+  source: dynamodb-list-tanñes
```
Comment (w0rmr1d3r) [inline stats_config.yaml:137]: I believe there's a typo here.
tables right?

Comment (roskyfrosky) [reply stats_config.yaml]: Yes, I have fixed it!

**Review by KevinHock — APPROVED** [review]
looks good to me, granted I didn't test the jq queries

**Review by w0rmr1d3r — APPROVED** [review]

**Comment by 0xdabbad00** [comment]
LGTM, thank you @roskyfrosky

**Pull request merged**